pattern-distributed/confs/sp-worker/conf/deployment.yaml

```diff
@@ -445,64 +383,59 @@ wso2.datasources:
         connectionTestQuery: SELECT 1
         validationTimeout: 30000
         isAutoCommit: false
-
   - name: EI_ANALYTICS
-    description: "The datasource used for EI Analytics dashboard feature"
+    description: The datasource used for EI Analytics dashboard feature
     jndiConfig:
       name: jdbc/EI_ANALYTICS
     definition:
       type: RDBMS
       configuration:
-        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/dashboard/database/EI_ANALYTICS;AUTO_SERVER=TRUE'
+        jdbcUrl: jdbc:h2:${sys:carbon.home}/wso2/worker/database/EI_ANALYTICS;AUTO_SERVER=TRUE
         username: wso2carbon
         password: wso2carbon
         driverClassName: org.h2.Driver
         maxPoolSize: 50
         idleTimeout: 60000
         validationTimeout: 30000
         isAutoCommit: false
-
-
   - name: IS_ANALYTICS_DB
-    description: "The data source used for WSO2 IS Analytics statistics"
+    description: The data source used for WSO2 IS Analytics statistics
     jndiConfig:
       name: jdbc/IS_ANALYTICS_DB
     definition:
       type: RDBMS
       configuration:
-        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/worker/database/IS_ANALYTICS_DB;AUTO_SERVER=TRUE'
+        jdbcUrl: jdbc:h2:${sys:carbon.home}/wso2/worker/database/IS_ANALYTICS_DB;AUTO_SERVER=TRUE
         username: wso2carbon
         password: wso2carbon
         driverClassName: org.h2.Driver
         maxPoolSize: 50
         idleTimeout: 60000
         validationTimeout: 30000
         isAutoCommit: false
-
   - name: GEO_LOCATION_DATA
-    description: "The data source used for geo location database"
+    description: The data source used for geo location database
     jndiConfig:
       name: jdbc/GEO_LOCATION_DATA
     definition:
       type: RDBMS
       configuration:
-        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/worker/database/GEO_LOCATION_DATA;AUTO_SERVER=TRUE'
+        jdbcUrl: jdbc:h2:${sys:carbon.home}/wso2/worker/database/GEO_LOCATION_DATA;AUTO_SERVER=TRUE
         username: wso2carbon
         password: wso2carbon
         driverClassName: org.h2.Driver
         maxPoolSize: 50
         idleTimeout: 60000
         validationTimeout: 30000
         isAutoCommit: false
-
   - name: APIM_ANALYTICS_DB
```

Comment:
If we are to clean the config file with unused configs, we can remove these datasources and other extensions used by different analytics solutions

Reply:
These configs come with the default pack, right?. Shouldn't we only change configuration relevant to deployment in the deafult config?